README.md

```diff
@@ -103,12 +103,12 @@ progress follow the [issue #471][17]
 
 #### Arch Linux
 
-We have a package in the AUR [here](https://aur.archlinux.org/packages/kubeone).
-Use your favorite method to build it on your system, for example by using
-`aurutils`:
-```bash
-aur sync kubeone && pacman -S kubeone
-```
+`kubeone` can be found in the official Arch Linux repositories:
+
+[https://www.archlinux.org/packages/community/x86_64/kubeone/](https://www.archlinux.org/packages/community/x86_64/kubeone/)
+
+Install it via `pacman -S kubeone`
```

Comment:
Tiny nit: I think we should leave this command in a code block to match the style used in the rest of README.

Reply:
@xmudrii good catch. Fixed.